appscale/tools/agents/azure_agent.py

```diff
@@ -1193,3 +1214,92 @@ def does_resource_group_exist(self, resource_group_name, resource_client):
     if resource_group_name in resource_group_names:
       return True
     return False
+
+  def create_image(self, instance_id, image_name, parameters, dest_container):
```

Comment:
Does this function belong in the tools? It seems like appscale-testing would be a better place for it since the user does not need to perform this operation.